basic/src/test/java/org/jboss/hal/testsuite/test/configuration/infinispan/InvalidationCacheTestCase.java

```diff
@@ -12,7 +12,7 @@
  * @author mkrajcov <mkrajcov@redhat.com>
  */
 @RunWith(Arquillian.class)
-@Category(Standalone.class)
+@Category(Shared.class)
```

Comment:
Added Shared, although the navigation for domain mode persisted from Standalone, so the navigation for Domain mode wont work...